### PR TITLE
UserStory16

### PR DESCRIPTION
**Pull request by AnjaSiewert**
[US16](https://github.com/AnjaSiewert/capstone_globehopper/issues/25)
--> Add Edit function to form on ⭐️ page
--> users can now mark countries as favorite ⭐️, fill out the form and edit their entries

### CODE REVIEW

File: components/Entry/index.js [hunk 1]
```diff
@@ -1,47 +1,236 @@
...
+
+          <StyledButton onClick={() => setIsEditing(!isEditing)}>
+            {isEditing ? "Cancel" : "Edit"}
+          </StyledButton>
+        </StyledList>
+
+        {isEditing && (
+          <>
+            <StyledForm onSubmit={handleSubmit} aria-label="Add own entries">
+              <h2>Edit my planed trip</h2>
```
Comment (salaos) [inline components/Entry/index.js]: typo: planned

File: pages/favorites.js [hunk 2]
```diff
@@ -21,15 +28,40 @@ export default function FavoriteCountriesPage({
     listFavoriteCountries.find((info) => info.name === country.name.common)
   );
 
+  function handleAddEntry(newEntry) {
+    setEntries([{ ...newEntry }, ...entries]);
```
Comment (salaos) [inline pages/favorites.js:32]: I think the spread operator for ...newEntry is not needed since newEntry is already an object

**Review by salaos — APPROVED** [review]
Added tiny improvements, LGTM! :)

File: pages/favorites.js (hunk 2 above)
Comment (AnjaSiewert) [reply pages/favorites.js:32]: Hi Nina, is actually doesn't work without the operator. So I kept it :)

**Pull request merged**